.github/workflows/pull_request.yml

```diff
@@ -34,16 +34,15 @@ jobs:
     steps:
       # :-( https://github.community/t/support-for-yaml-anchors/16128
       - {uses: actions/checkout@v2, with: {fetch-depth: 0}}
-      - uses: actions/setup-python@v2
+      - run: python --version
```

Comment:
Thanks for solving that mystery. Could you submit this (and maybe others?) separately so we expedite it?

Reply:
I have cherry-picked these commits to <https://github.com/thesofproject/sof-test/pull/765>.

Reply:
Thanks! Approved.

Here's a git trick I use all the time:
```shell
git push my_github_fork HEAD~2:refs/heads/actions-python
```
The trick is `HEAD~2`, so you don't need to switch branch or cherry pick anything.